views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -53,6 +53,9 @@ Cypress.Commands.add('importTestPackage', (fileContent, fileName) => {
         }
     );
 
+    // windows workaround
+    cy.wait(2000);
```

Comment:
See previous comment. If there is no network request to wait for, try to do an assertion about the expected element. We can use  `.wait(2000)` if we have to, but it should be the last resort.

Reply:
change for the proper request `cy.wait('@fileUpload');`